.gitignore

```diff
@@ -58,3 +58,38 @@ ssl/
 
 # Code editors
 .vscode
+.idea/**/workspace.xml
```

Comment:
It will be a good idea to just put `.idea/` in the gitignore rather than specifying the path of each of the files/folders inside that directory.

Reply:
Okay I will do it. This is me first time resolving a PR.. actually I looked at the JetBrains github, and they gave all these https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore which covers all the IDE's.. even WebStorm

Reply:
There has been no response. Is this PR reviewed yet?